content/Mobile_Dev/WebAuthn/index.adoc

```diff
@@ -0,0 +1,22 @@
+== WebAuthn development for mobile devices
+At this stage you have selected WebAuthn as your primary MFA option for your mobile solution. Through the rest of this series we are going to cover a variety of different steps and concepts that will help you along your implementation journey. Don’t be frightened if you are not familiar with WebAuthn, we are going to begin this section by giving an overview on what WebAuthn is, along with more resources for you to take a deeper dive into the concepts, and a how-to on deploying a sample application. Once you feel ready then proceed to the following steps where we will dive into OS and ecosystem specific concepts. 
+
+=== What if I'm not familiar with WebAuthn?
+If you are not familiar with WebAuth, and are looking to dive into the vast array of concepts, then please use our link:/Mobile_Dev/WebAuthn/WebAuthn_Primer.html[WebAuthn primer for mobile development]. This will provide a variety of resources including: links to helpful documentation, considerations for mobile development, and a guide for deploying your own sample application.
+
+=== Do I need OS specific client applications when my users are using iOS vs Android?
```

Comment:
Unless the page as a FAQ section, consider changing section titles from questions to statements that help direct the flow through the learning objectives.

Reply:
Changed in latest commit